### PR TITLE
Improve minor feature for repeat range for issue #985

### PR DESCRIPTION
**Pull request by fnzainal**
Just make a little change for my issue #985 .
Sometimes for helping memorize Quran need more repeat range, so I add 4,5,10,15,20 for this issue.
Hope it helpful for me and others. 

Thanks. Baarakallahufiikum

### CODE REVIEW

**Comment by ozbek** [comment]
This is essentially same as the existing user experience (albeit 8 options instead of 4), still limited options.

**Comment by ahmedre** [comment]
I agree with @ozbek - this isn’t great for ui and still doesn’t fully solve the problem. Let’s try to think of a better solution - maybe an “other” option that lets the person choose how many times?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -336,6 +336,11 @@
     <item>1 time</item>
     <item>2 times</item>
     <item>3 times</item>
+    <item>4 times</item>
+    <item>5 times</item>
+    <item>10 times</item>
+    <item>15 times</item>
+    <item>20 times</item>
```
Comment (ahmedre) [inline app/src/main/res/values/strings.xml:343]: can you add these to the other translation files?

File: app/src/main/java/com/quran/labs/androidquran/ui/fragment/AyahPlaybackFragment.java [hunk 2]
```diff
@@ -245,16 +245,29 @@ private void updateEnforceBounds(int rangeRepeatPosition) {
...
-      return repeat;
+    switch (repeat) {
+      case -1:
+        return REPEAT_MAX;
+      case 9:
+        return 5;
+      case 14:
+        return 6;
+      case 19:
+        return 7;
```
Comment (ahmedre) [inline app/src/main/java/com/quran/labs/androidquran/ui/fragment/AyahPlaybackFragment.java:256]: why?

File: app/src/main/java/com/quran/labs/androidquran/ui/fragment/AyahPlaybackFragment.java [hunk 3]
```diff
@@ -245,16 +245,29 @@ private void updateEnforceBounds(int rangeRepeatPosition) {
...
 
   private int positionToRepeat(int position) {
     if (position >= REPEAT_MAX) {
       return -1;
+    } else if (position == 5){
+      return 9;
+    } else if (position == 6){
+      return 14;
+    } else if (position == 7){
+      return 19;
```
Comment (ahmedre) [inline app/src/main/java/com/quran/labs/androidquran/ui/fragment/AyahPlaybackFragment.java:270]: do we need this?

**Comment by ahmedre** [comment]
though i'd love a better solution, i am ok taking this if you can help clean it up - jazakumAllah khairan for your patience and for your help with this.

**Comment by ahmedre** [comment]
also can we just make the new options 5 and 10?

**Comment by benomaire** [comment]
Can we use a [horizontal number picker](https://github.com/ShawnLin013/NumberPicker) in the settings? and only add 5 and 10 for example for the lower icon as @ahmedre suggested?

**Comment by benomaire** [comment]
@fnzainal Do you allow me to carry on this work?

**Comment by ahmedre** [comment]
closing now since @benomaire finished it in #1519

**Comment by ahmedre** [comment]
jazakumAllah khairan!

**Pull request closed without merging**